diff --git a/dist/unstable/rpc/RpcWorker.js b/dist/unstable/rpc/RpcWorker.js
index d84189dd147f0667b6a14ff592ef5d825916f30c..f503a554937d91f2a0ccd6a0619a182ed5e03577 100644
--- a/dist/unstable/rpc/RpcWorker.js
+++ b/dist/unstable/rpc/RpcWorker.js
@@ -8,7 +8,7 @@ import * as Transferable from "../workers/Transferable.js";
  * @category initial message
  */
 export class InitialMessage extends /*#__PURE__*/ServiceMap.Service()("effect/rpc/RpcWorker/InitialMessage") {}
-const ProtocolTag = /*#__PURE__*/ServiceMap.Service("@effect/rpc/RpcServer/Protocol");
+const ProtocolTag = /*#__PURE__*/ServiceMap.Service("effect/rpc/RpcServer/Protocol");
 /**
  * @since 4.0.0
  * @category initial message
